### DISCUSSION TITLE
Update?

### DISCUSSION THREAD

**Discussion by prolife86** [gsuberland/open-network-ms#3] [General]
Hi gsuberland,

How are you doing?
Was Spork revision 1 a success?

Do you have any updates in regards to this project?

**Comment by gsuberland** [comment]
Nope, never got anywhere with it. Life got in the way and working on a
prototype when you've only got one UPS and it's in production is not really
viable.

**Reply by prolife86** [reply]
Understandable, thank you for the update! :)
